src/DGtal/geometry/curves/Naive3DDSSComputer.h

```diff
@@ -0,0 +1,314 @@
+/**
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Lesser General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or  (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ **/
+
+#pragma once
+
+/**
+ * @file Naive3DDSSComputer.h
+ * @author Kacper Pluta (\c kacper.pluta@esiee.fr )
+ * Laboratoire d'Informatique Gaspard-Monge - LIGM, A3SI, France
+ *
+ * @date 2014/10/07
+ *
+ * Header file for module Naive3DDSSComputer.cpp
+ *
+ * This file is part of the DGtal library.
+ */
+
+#if defined(Naive3DDSSComputer_RECURSES)
+#error Recursive header files inclusion detected in Naive3DDSSComputer.h
+#else // defined(Naive3DDSSComputer_RECURSES)
+/** Prevents recursive inclusion of headers. */
+#define Naive3DDSSComputer_RECURSES
+
+#if !defined Naive3DDSSComputer_h
+/** Prevents repeated inclusion of headers. */
+#define Naive3DDSSComputer_h
+
+//////////////////////////////////////////////////////////////////////////////
+// Inclusions
+#include <iostream>
+#include <list>
+#include "DGtal/base/Exceptions.h"
+#include "DGtal/base/Common.h"
+#include "DGtal/kernel/PointVector.h"
+#include "DGtal/kernel/CInteger.h"
+#include "DGtal/geometry/curves/ArithmeticalDSSComputer.h"
+#include "DGtal/base/ConstIteratorAdapter.h"
+#include "DGtal/kernel/BasicPointFunctors.h"
+//////////////////////////////////////////////////////////////////////////////
+
+namespace DGtal
+{
+
+/////////////////////////////////////////////////////////////////////////////
+// class Naive3DDSSComputer
+/**
+ * Description of class 'Standard3DDSSComputer' <p>
+ * \brief Aim:
+ * Dynamic recognition of a 3d-digital straight segment (DSS)
+
```

Comment:
Is Naive3DDSSComputer a model of some Computer concept ?
http://dgtal.org/doc/nightly/moduleGridCurveAnalysis.html
... for instance the CDynamicBidirectionalSegmentComputer
I would suggest to add this in the doc, as well as models of these concepts.